common.sh

```diff
@@ -294,7 +295,11 @@ create_instance()
     if [[ -n ${USER_DATA_FILE} && -f ${USER_DATA_FILE} ]]; then
         addl_args+=" --user-data file://${USER_DATA_FILE}"
     fi
-
+    # NVIDIA drivers and CUDA toolkit are large, allocate more EBS space for them.
+    if [ "$ami_arch" = "x86_64" ]; then
```

Comment:
Does this have to be conditional on the architecture? I don't know what our current usage is like, but I've run into a wall with 32G volumes even with a handful installations from our test suite when not using a shared-FS. Might as well do this for all instances.

Reply:
The default EBS volume size is 16G. Our current workload on ARM test does not require extra volume, so I added a condition here to make the volume increase well motivated.

Reply:
Alright.